### DISCUSSION TITLE
Welcome to emergent Discussions!

### DISCUSSION THREAD

**Discussion by rcoreilly** [emer/emergent#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by StephenJRead** [comment]
Stephen J. Read, Mendel B. Silberberg Professor of Social Psychology
3620 McClintock Ave, 501 SGM
University of Southern California
Los Angeles, CA 90089-1061
read@usc.edu<mailto:read@usc.edu>

On Dec 20, 2020, at 12:23 AM, rcoreilly <notifications@github.com<mailto:notifications@github.com>> wrote:


👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  *   Ask questions you’re wondering about.
  *   Share ideas.
  *   Engage with other community members.
  *   Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/emer/emergent/discussions/68__;!!LIr3w8kk_Xxm!4KA7PySHDmmrQ_ER_yUz8NY-Jf0DGWVeDe2FV5uLX_4hL5PjHmGDVaRra5U$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AFASMSZ3J6BW4EZYHTIZZEDSVWX7FANCNFSM4VC5HTRQ__;!!LIr3w8kk_Xxm!4KA7PySHDmmrQ_ER_yUz8NY-Jf0DGWVeDe2FV5uLX_4hL5PjHmGDj8Ft77k$>.

I’m Steve Read, Professor of Social Psychology and Brain and Cognitive Sciences at USC.  I’m a social psychologist and cognitive scientist by training and I do computational modeling of personality, motivation, social reasoning and social perception.  I have been doing computational modeling with Randy’s software since the PDP++ days around 2000 or so. I have also been teaching neural network modeling with emergent scene around 2010.
